Change/Move pagespeed cache to new redis DB

cru-Luis-Rodriguez opened the pull request:
Depends on the ecs_config change to add new ENV for Redis database. 
https://github.com/CruGlobal/ecs_config/pull/295

cru-Luis-Rodriguez committed:
Change/Move pagespeed cache to new redis DB

twinge commented on main/Dockerfile:
```diff
@@ -18,5 +19,5 @@ COPY .htpasswd /usr/local/openresty/nginx/conf/.htpasswd
 RUN envsubst '$RESOLVER $S3_ADDR $STORYLINES_ADDR' < /usr/local/openresty/nginx/conf/location.conf.t > /usr/local/openresty/nginx/conf/location.conf \
   && rm /usr/local/openresty/nginx/conf/location.conf.t
-RUN envsubst '$DOMAIN $PROXY_ADDR $REDIS_PORT_6379_TCP_ADDR_A' < /usr/local/openresty/nginx/conf/pagespeed-site.conf.t > /usr/local/openresty/nginx/conf/pagespeed-site.conf \
+RUN envsubst '$DOMAIN $PROXY_ADDR $REDIS_PORT_6379_TCP_ADDR_A $REDIS_PORT_6379_TCP_ADDR_PS' < /usr/local/openresty/nginx/conf/pagespeed-site.conf.t > /usr/local/openresty/nginx/conf/pagespeed-site.conf \
```
do you still need REDIS_PORT_6379_TCP_ADDR_A now that you have REDIS_PORT_6379_TCP_ADDR_PS?

cru-Luis-Rodriguez replied:
I'll update the PR, I just left it there cus' Bryn mentioned that the "$REDIS_PORT_6379_TCP_ADDR_A" EVN variable might be used somewhere else on the build or in another process of the cruorg proxy. 

Granted this particular line of code does not need it.

cru-Luis-Rodriguez committed:
Remove ENV from envsubst

Remove $REDIS_PORT_6379_TCP_ADDR_A from envsubst command. The variable has been replaced with  $REDIS_PORT_6379_TCP_ADDR_PS.

cru-Luis-Rodriguez commented:
@twinge can you review?  I made the change you requested.
